opennms-base-assembly/src/main/filtered/bin/find-java.sh

```diff
@@ -50,11 +50,11 @@ get_java_version_string() {
 get_real_path() {
 	file_to_find="$1"
 	readlink="$(command -v readlink 2>/dev/null)"
```

Comment:
Could move this out of this function since we don't need to recursively set this